src/components/CommentReplyPreview/CommentReplyPreview.tsx

```diff
@@ -7,55 +7,70 @@ import { neutral, space, palette } from "@guardian/src-foundations";
 import { Pillar, CommentType } from "../../types";
 
 const commentControlsButton = (pillar: Pillar) => css`
-  font-weight: bold;
+  ${textSans.small()};
   margin-right: ${space[2]}px;
   color: ${palette[pillar][400]};
   border: 0;
 `;
 
 const replyWrapperStyles = css`
+  padding-top: 5px;
+  padding-bottom: 5px;
   display: flex;
   flex-direction: row;
-  padding: 5px;
+  align-items: center;
 `;
 const replyArrowStyles = css`
   fill: grey;
-  padding: 5px;
+  padding-top: 2px; /* TODO: find a better solution */
```

Comment:
`find a better solution` - it's more usefule if the comment explains what the problem is with using padding and why it had to be used anyway